pandajedi/jediorder/TaskRefiner.py

```diff
@@ -230,7 +230,13 @@ def runImpl(self):
                         if taskParamMap.get("inputPreStaging") and taskParamMap.get("taskType") == "anal" and taskParamMap.get("prodSourceLabel") == "user":
                             tmpLog.info("checking about data carousel")
                             try:
-                                prestaging_list, ds_list_dict = data_carousel_interface.get_input_datasets_to_prestage(jediTaskID, taskParamMap)
+                                try:
+                                    # check input datasets to prestage
+                                    prestaging_list, ds_list_dict = data_carousel_interface.get_input_datasets_to_prestage(jediTaskID, taskParamMap)
+                                except Exception as e:
+                                    # got error (e.g. due to DDM error); skip and retry in next cycle
+                                    tmpLog.error(f"failed to check input datasets to prestage ; got {e} ; skip and retry next time")
+                                    continue
```

Comment:
@tmaeno Is it correct to put "continue" in TaskRefiner like this to let it retry the task over temporary (DDM) issue?

Reply:
I think so.